mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -218,4 +218,28 @@ public void reasonableLongTermShouldNotBeIgnored() throws Exception {
         assertThat(messageManager.search(new SearchQuery(SearchQuery.bodyContains(reasonableLongTerm)), session))
             .containsExactly(composedMessageId.getUid());
     }
+
+    @Test
```

Comment:
Not failing, it doesn't index on that field, so doing a search of mails only on that field returns only the first message, so the assertion doesn't fail... so why `@ignore`?

Reply:
Or do you expect it to work, so I should add the uid of the second message in the assertion and then it will fail, thus the `@ignore` tag?

Reply:
> Or do you expect it to work, so I should add the uid of the second message in the assertion and then it will fail, thus the @ignore tag?

Of course, we expect the second message being indexed as well. And that is what should be tested. Current behaviour you demonstrated is buggy and should be treated as such.